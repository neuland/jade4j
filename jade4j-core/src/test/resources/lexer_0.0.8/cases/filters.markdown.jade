html
  body
    :markdown
      This is _some_ awesome **markdown**
      whoop.