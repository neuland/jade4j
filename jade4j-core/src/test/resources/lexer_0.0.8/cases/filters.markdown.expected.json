{"type":"tag","line":1,"val":"html","selfClosing":false}
{"type":"indent","line":2,"val":2}
{"type":"tag","line":2,"val":"body","selfClosing":false}
{"type":"indent","line":3,"val":4}
{"type":"filter","line":3,"val":"markdown"}
{"type":"start-pipeless-text","line":3}
{"type":"text","line":4,"val":"This is _some_ awesome **markdown**"}
{"type":"newline","line":5}
{"type":"text","line":5,"val":"whoop."}
{"type":"end-pipeless-text","line":5}
{"type":"outdent","line":5}
{"type":"outdent","line":5}
{"type":"eos","line":5}